templates/_macros/quote-wrapper.jinja

```diff
@@ -0,0 +1,178 @@
+      </i>
+    </p>
+  {%- endmacro %}
+
+  {%- macro _citation_block() -%}
+    {%- if has_citation -%}
+      {#- Source name and title/org must be wrapped in grid if they are both present, for responsive view -#}
+      {%- macro _citation_block_source_primary() -%}
+        {% if has_citation_source_name -%}
+          <!--Optional citation source name-->
+          <h5 class="u-no-margin--bottom">{{ citation_source_name_text }}</h5>
+        {%- endif %}
+      {%- endmacro -%}
+
+      {%- macro _citation_block_source_secondary() -%}
+        {% if has_citation_source_title or has_citation_source_organisation -%}
+          <p class="u-text--muted">
+            {% if has_citation_source_title -%}
+              <!--Optional citation source title-->
+              <span>{{ citation_source_title_text }}</span>
+              <br>
+            {%- endif %}
+            {% if has_citation_source_organisation -%}
+              <!--Optional citation source organisation-->
+              <span>{{ citation_source_organisation_text }}</span>
+            {%- endif %}
+          </p>
+        {%- endif %}
+      {%- endmacro -%}
+      
+      <!--Optional citation block-->
+      <hr class="u-hide--large is-muted">
+      {%- if has_citation_source_name and (has_citation_source_title or has_citation_source_organisation) %}
+        <!-- When source name and title/org are both present, wrap them in a 50/50 row to split them on medium -->
+        <div class="row">
+          <div class="col-3 col-medium-3">
+            {{ _citation_block_source_primary() }}
+          </div>
+          <div class="col-3 col-medium-3">
+            {{ _citation_block_source_secondary() }}
+          </div>
+        </div>
+      {%- else -%}
+        {{ _citation_block_source_primary() }}
+        {{ _citation_block_source_secondary() }}
+      {%- endif -%}
+    {%- endif -%}
+  {%- endmacro %}
+
+  {%- macro _heading_block() %}
+    {% if has_heading_row -%}
+      <!--Optional heading -->
+      <div class="p-section--shallow">
+        <hr class="is-highlighted is-fixed-width">
+        <div class="row">
+          {%- if has_title %}
+            <!--Optional heading text-->
+            <div class="col-3 col-medium-3">
+              <div class="p-section--shallow">
+                <p class="p-muted-heading">{{ title_text }}</p>
+              </div>
+            </div>
+          {%- endif -%}
+
+          {%- if has_heading_link %}
+            <!--Optional heading link -->
+            <div class="col-3 col-medium-3 col-start-large-10 col-start-medium-5">
+              <div class="p-section--shallow">
+                <p class="p-text--default">
+                  {{ heading_link_content }}
+                </p>
+              </div>
+            </div>
+          {% endif -%}
+        </div>
+      </div>
+    {% endif -%}
+  {%- endmacro %}
+
+  <div class="p-section">
+    {{- _heading_block() -}}
+    <div class="row">
```

Comment:
@bartaz  When I use `row--25-75`, the nested grid for the source is broken:

Expected:
![Screenshot from 2024-08-27 11-42-11](https://github.com/user-attachments/assets/d10ecffe-6867-4195-9cb9-da92644e3c0a)

Observed:
![Screenshot from 2024-08-27 11-41-56](https://github.com/user-attachments/assets/bfa73c4f-9d9a-4f40-9235-1af4a7ccb1cc)

Reply:
Update: Lyubo's made a [design simplification](https://www.figma.com/design/vH19y2J2YqwUfksebSUvTW?node-id=26-1135&m=dev#923221984) that means this can be done

Reply:
Switched to 25/75 row